auth-api/src/auth_api/models/entity.py

```diff
@@ -41,6 +41,7 @@ class Entity(BaseModel):  # pylint: disable=too-few-public-methods
 
     contacts = relationship('ContactLink', back_populates='entity')
     corp_type = relationship('CorpType', foreign_keys=[corp_type_code], lazy='joined', innerjoin=True)
+    affiliations = relationship('Affiliation', cascade='all,delete,delete-orphan', lazy='select')
```

Comment:
This must be from what he was working on, I have rebased the code in between my development. I think these backend changes coming from that. 
I think it will go away once patrick is done with his changes